### DISCUSSION TITLE
Release Date for the Android Version

### DISCUSSION THREAD

**Discussion by maxii1996** [2fast-team/2fast#155] [Q&A]
Hello,

I've been following this project for a while and even supported the app by purchasing the full version. So far, I’m very satisfied with its performance. However, I've been waiting for the Android version for some time now, and I would like to have it available so I don't have to rely entirely on my PC.

![20240810_133451](https://github.com/user-attachments/assets/5701cbac-c1e9-4480-9eef-fec4ee57362c)

Thank you in advance.

Best regards,

**Comment by jp-weber** [comment]
Thank you very much for your support. The Android version is almost feature complete but I want to fine tune the performance so that there is no negative experience when it is released, so I can't give a release date yet.

However, I will publish the next Android preview (3) version soon.
